testsuite/cli/src/client_proxy.rs

```diff
@@ -739,6 +739,7 @@ impl ClientProxy {
                 .get_txn_by_acc_seq(account, sequence_number - 1, true)
             {
                 Ok(Some(txn_view)) => {
+                    println!();
```

Comment:
maybe it'll be better to add the extra line after the actual "..." output here? https://github.com/libra/libra/blob/master/testsuite/cli/src/client_proxy.rs#L724-L759

Reply:
Add this is because we have following message "Transaction executed", without this, it looks like this:

```
..................................transaction executed!
```
After added this println:
```
..................................
transaction executed!
```